tests/spread/plugins/plainbox/run/task.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Build and run a basic checkbox snap
+
+systems: [ubuntu-16*]
+
+environment:
+  SNAP_DIR: ../snaps/checkbox
+
+restore: |
+  cd "$SNAP_DIR"
+  snapcraft clean
+  rm -f ./*.snap
```

Comment:
are you uninstalling the snap in other place? otherwise you should do it here

Reply:
good catch, we are not doing this.